src/controllers/favorites.js

```diff
@@ -127,6 +127,14 @@ function getSections() {
         overlayPlayButton: true,
         overlayText: false,
         centerText: true
+    }, {
+        name: 'Live TV Channels',
```

Comment:
This should be a translation key for localization:
```suggestion
        name: 'Channels',
```

Reply:
The name for "Videos" is invalid also if you want to fix it also... it should be "HeaderVideos". :slightly_smiling_face: